components/script/dom/node.rs

```diff
@@ -290,6 +294,7 @@ impl Node {
             node.set_flag(NodeFlags::IS_IN_DOC, parent_in_doc);
             node.set_flag(NodeFlags::IS_IN_SHADOW_TREE, parent_in_shadow_tree);
             node.set_flag(NodeFlags::IS_CONNECTED, parent_is_connected);
+            node.set_flag(NodeFlags::IS_USER_AGENT_WIDGET, parent_is_user_agent_widget);
```

Comment:
It doesn't make sense for this flag to be propagated to children, does it? What would you need that for?